example_code/applications/english_auction/Cargo.toml.bak

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "stylus-auction-example"
```

Comment:
```suggestion
name = "stylus-arrays-example"
```

Reply:
Oh, this one is in `English Auction`, so the current one is correct, we don't need to change, I will make a pr to change it back